dot/parachain/availability-store/availability_store.go

```diff
@@ -36,142 +79,318 @@ type AvailabilityStoreSubsystem struct {
 	SubSystemToOverseer chan<- any
 	OverseerToSubSystem <-chan any
 	availabilityStore   AvailabilityStore
+	pruningConfig       pruningConfig
+	clock               subsystemClock
 	//TODO: pruningConfig PruningConfig
 	//TODO: clock         Clock
 	//TODO: metrics       Metrics
 }
 
 // AvailabilityStore is the struct that holds data for the availability store
 type AvailabilityStore struct {
```

Comment:
Why is this structure exported, and why are all its fields and methods not exported?

Reply:
Go point, I've changed this to be unexported.